readme.md

```diff
@@ -16,8 +16,10 @@ touch /path/to/squabble/storage/database.sqlite
 php artisan migrate
 ```
 
-**Important:** Your site won't work without a `.env` file in the root of the repo. Please reference `env.example` for the contents of the `.env` file.
-
+**Important:**
+ - Your site won't work without a `.env` file in the root of the repo. Please reference `env.example` for the contents of the `.env` file.
+ - Make sure your .env and database.sqlite files are named exactly that.
```

Comment:
@flowersw can you make this a little clearer. What is "exactly that"? Also. You should probably use the ticks like the other lines do to show it more properly in markdown.